wetmap/src/components/pages/tutorial/.stories.tsx

```diff
@@ -0,0 +1,28 @@
+// More on how to set up stories at: https://storybook.js.org/docs/writing-stories#default-export
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import TutorialPageView from './view';
+
+
+const meta: Meta<typeof TutorialPageView> = {
```

Comment:
Silly ask but can you please change references to "tutorial" to "onboarding"?
This particular pieces is for onboarding new users  and is not a tutorial/guide. We will make new guides/tutorials much later on

Reply:
This will be part of another ticket. In this case this is example only